trie-db/src/triedb.rs

```diff
@@ -95,7 +95,7 @@ where
 	/// may require a database lookup. If `is_root_data` then this is root-data and
 	/// is known to be literal.
 	fn get_raw_or_lookup(&'db self, node: &[u8], partial_key: &[u8]) -> Result<Cow<'db, DBValue>, H::Out, C::Error> {
```

Comment:
i think it could be helpful to have a doc here about `partial_key` and how it's expected to be an encoded nibble-slice.